src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
+// #region page
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  font-weight: 400;
+
+  --border-color: rgb(243, 243, 243);
+  --blue-color: rgb(0, 172, 220);
+  --dark-blue-color: rgb(6, 11, 53);
+  --grey-color: rgb(97, 96, 112);
+  --white-color: rgb(255, 255, 255);
 }
+
+// #endregion
+
+// #region card
+.card {
+  display: block;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid var(--border-color);
+
+  &__product {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    padding: 32px 16px 16px;
+  }
+
+  &__images img {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__content {
+    margin-top: 40px;
+  }
+}
+
+// #endregion
+
+// #region content
+.name {
+  margin-bottom: 4px;
+  color: var(--dark-blue-color);
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  color: var(--grey-color);
+  line-height: 14px;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__stars {
+    .stars {
+      display: flex;
+
+      &__star {
+        width: 16px;
+        height: 16px;
+        display: block;
+        background-image: url(../images/star.svg);
+        background-position: center;
+        background-repeat: no-repeat;
+        margin-right: 4px;
+      }
+
+      &__star:nth-child(5) {
+        margin-right: 0;
+      }
+
+      &--4 .stars__star:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: var(--dark-blue-color);
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--grey-color);
+  }
+
+  &__current {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: var(--dark-blue-color);
+  }
+}
+
+// #endregion
+
+// #region button
+button {
+  width: 166px;
+  height: 40px;
```

Comment:
Use padding instead of hardcode the height
```suggestion
```

Reply:
I'll leave the code as it is, otherwise it breaks, or show me how to fix it.